Not compatible with ember-source

Ember is now distributed via npm and no longer exists in bower_components. 

```
Cannot find module '/Users/fabian/Projects/.../bower_components/ember/ember-template-compiler.js'
Error: Cannot find module '/Users/fabian/Projects/.../bower_components/ember/ember-template-compiler.js'
    at Function.Module._resolveFilename (module.js:469:15)
    at Function.Module._load (module.js:417:25)
    at Module.require (module.js:497:17)
    at require (internal/module.js:20:19)
    at Class.htmlbarsOptions (/Users/fabian/Projects/.../node_modules/ember-cli-data-export/node_modules/ember-cli-htmlbars/ember-addon-main.js:99:25)
```

Looks like ember-cli-htmlbars needs an update.

--- a/package.json
+++ b/package.json
@@ -14,22 +14,21 @@
   },
   "repository": "https://github.com/roofstock/ember-cli-data-export",
   "engines": {
-    "node": ">= 0.12.0"
+    "node": ">= 4"
   },
-  "author": "Matt Armstrong",
   "license": "MIT",
   "devDependencies": {
     "broccoli-asset-rev": "^2.4.5",
     "ember-ajax": "^2.4.1",
-    "ember-cli": "2.11.1",
+    "ember-cli": "^2.12.0",
     "ember-cli-app-version": "^2.0.0",
     "ember-cli-dependency-checker": "^1.3.0",
-    "ember-cli-htmlbars": "^1.1.1",
+    "ember-cli-htmlbars": "^1.2.0",
     "ember-cli-htmlbars-inline-precompile": "^0.3.6",
     "ember-cli-inject-live-reload": "^1.4.1",
     "ember-cli-jshint": "^2.0.1",
     "ember-cli-qunit": "^3.0.1",
-    "ember-cli-release": "^0.2.9",
+    "ember-cli-release": "1.0.0-beta.2",
     "ember-cli-shims": "^1.0.2",
     "ember-cli-sri": "^2.1.0",
     "ember-cli-test-loader": "^1.1.0",
@@ -38,8 +37,8 @@
     "ember-disable-prototype-extensions": "^1.1.0",
     "ember-export-application-global": "^1.0.5",
     "ember-load-initializers": "^0.6.0",
-    "ember-resolver": "^2.0.3",
-    "ember-source": "~2.11.0",
+    "ember-resolver": "^3.0.0",
+    "ember-source": "~2.12.0",
     "loader.js": "^4.0.10"
   },
   "keywords": [
@@ -50,10 +49,10 @@
     "csv"
   ],
   "dependencies": {
-    "ember-cli-htmlbars": "^1.1.1",
+    "ember-cli-htmlbars": "^1.2.0",
     "ember-cli-babel": "^5.1.7",
     "ember-select-list": "^0.9.5",
-    "xlsx": "^0.8.0"
+    "xlsx": "^0.9.0"
   },
   "ember-addon": {
     "configPath": "tests/dummy/config"

